index.html

```diff
@@ -487,7 +487,17 @@ <h3>
             "<dfn data-dfn-for="text-direction">auto</dfn>" (default)
           </dt>
           <dd>
-            No explicit directionality.
+            <p>
+              Direction determined from content using the first-strong part of
```

Comment:
How about we do: 

```suggestion
              Direction determined from content using <a data-cite="bidi#P2">Rule P2</a> of
```

That has the